frontend/src/lib/candidate/components/candidateContext/RequireLogin.svelte

```diff
@@ -33,7 +39,15 @@ Require candidates to be logged in to view the children of this component.
 -->
 
 {#if $userStore}
-  <slot />
+  {#if candidate != undefined}
+    <slot />
+  {:else}
+    <BasicPage title="">
```

Comment:
May not be necessary but perhaps some title, even just "Error" or something, could be added?